tcms/issuetracker/redmine_integration.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+    Helper which facilitate actual communications with JIRA.
+"""
+
+import warnings
+import threading
+
+
+class RedmineThread(threading.Thread):
+    """
+        Execute REDMINE REST API code in a thread!
+
+        Executed from the IssueTracker interface methods.
+    """
+
+    def __init__(self, rpc, testcase, bug):
+        """
+            @rpc - REDMINE Python/API object
+            @testcase - TestCase object
+            @bug - Bug object
+        """
+
+        self.rpc = rpc
+        self.testcase = testcase
+        self.bug = bug
+        super(RedmineThread, self).__init__()
```

Comment:
In Python 3 this is just `super()`. We really need to teach pylint about this (-;

Reply:
OK